applications/common/backend/src/main/resources/libs/kyanite/common/components/common/template/.content.json

```diff
@@ -0,0 +1,11 @@
+{
+  "sling:resourceType": "wcm/dialogs/components/select",
+  "name": "templatePath",
+  "label": "Layout",
+  "default": {
+    "sling:resourceType": "wcm/dialogs/components/select/selectitem",
+    "label": "Default",
+    "value": "./templates/template-default.html",
+    "selected": true
```

Comment:
I think you could remove it, as it's not supported anymore

Reply:
Actually, it is. This is a dialog field for cases when we want to have a choice of layout templates in a component - it's an optional part of the [templateHandler contract](https://github.com/websight-io/kyanite/pull/453). It is here, because I started this branch from the contract branch.